index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="desk.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <nav id="navbar">
+        <ul id="list">
+            <li><a id="home" href="index.html">Home</a></li>
+            <li><a id="portfolio" href="portfolio.html">Portfolio</a></li>
+            <li><a id="about" href="about.html">About</a></li>
+            <li><a id="contact" href="contact.html">Contact</a></li>
+            <li><img id="line" src="./images/Rectangle_52.png" alt="rectangle to underline options"></li>
+        </ul>
+    </nav>
+
+    <main>
+        <section id="presentation">
+            <article>
+                <p id="name">Francisco Borja</p>
```

Comment:
- [ ] Please make sure that main title (e.g., "Francisco Borja ...") is a `<h1>` tag instead of paragraph`<p>`
 Kindly use this LINK learn more about [Semantic HTML](https://www.w3docs.com/learn-html/semantic-elements-in-html5.html)

Reply:
not resolved